TACC/Core-Portal
# Task/TV3-57: Replace JobSubmission model with tags

## Overview
Replaces the use of `JobSubmission` to keep track of portal-specific jobs with tags.

## Related

* [TV3-57](https://jira.tacc.utexas.edu/browse/TV3-57)

## Changes
Replaced JobSubmission with tags on job get.
Filled out tag field on job post.

## Testing
1. Submit a job with the tag `CEP` (whatever `_PORTAL_NAMESPACE` is set as locally)
2. Have other jobs submitted with a different tag (or no tag)
3. Ensure only the jobs submitted with `CEP` tag shows up in https://cep.test/workbench/history/jobs/

## Notes
1. Wasn't sure if we could remove the `JobSubmission` & the `import-jobs` command (which relies on JobSubmission).
2. Also haven't tested with complete v3 job post. So testing step assumes manual job submission with `tapipy`.
